composer.lock

```diff
@@ -348,25 +348,28 @@
         },
         {
             "name": "myclabs/deep-copy",
-            "version": "1.7.0",
+            "version": "1.8.1",
```

Comment:
v.1.8.1 breaks compatibility with php 7.0, would suggest to go for a lower version instead

Reply:
True. Sorry, my bad.
I will cancel this PR and drop a new one